app/scripts/modules/amazon/serverGroup/configure/serverGroupConfiguration.service.js

```diff
@@ -73,6 +73,31 @@ module.exports = angular.module('spinnaker.aws.serverGroup.configure.service', [
         }
       };
 
+      command.blockDeviceMappingsSource = () => {
```

Comment:
I didn't see anything leaking into the pipeline json or over the wire ... we've got other functions being added to `command` so I don't anticipate an issue.